cmd/flags.go

```diff
@@ -214,6 +214,10 @@ func clientFlags() []cli.Flag {
 			Name:  "subdir",
 			Usage: "mount a sub-directory as root",
 		},
+		&cli.StringFlag{
+			Name:  "atime-mode",
```

Comment:
set `Value: "noatime"`, which indicates the default value.

Reply:
Fixed